src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -681,7 +681,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Dim parameter As IParameterSymbol = boundParameterEqualsValue.Parameter
             Dim value As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundParameterEqualsValue.Value))
             Dim kind As OperationKind = OperationKind.ParameterInitializerAtDeclaration
-            Dim isInvalid As Boolean = DirectCast(boundParameterEqualsValue.Value, IOperation).IsInvalid
+            Dim isInvalid As Boolean = value.Value.IsInvalid
```

Comment:
sure, I can do that.

FYI. in a few days, I will remove all isInvalid as we discussed in the earlier meeting.